source/docs/api-reference/simulation/simulation-intro.rst

```diff
@@ -73,6 +73,8 @@ Inputs and Outputs
 
 All ``SimState`` objects contain multiple inputs to manipulate the state of the device based on simulation physics calculations. For example, all device ``SimState`` objects have a supply voltage input:
 
+.. important::  Non-FRC platforms are required to set supply voltage, as this affects simulation calculations. It's recommended that FRC users set supply voltage to ``getBatteryVoltage()`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/RobotController.html#getBatteryVoltage()>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_robot_controller.html#a4b1e42e825583c82664a4ecc5d81b83f>`__) to take advantage of WPILib's ``BatterySim`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/simulation/BatterySim.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1sim_1_1_battery_sim.html>`__).
```

Comment:
- "as this affects" -> "as it affects"
- Same as in motor controller integration, "WPILib's BatterySim API (...)."